public/locales/en/dashboard.json

```diff
@@ -1,3 +1,34 @@
 {
-  "in-progress": "In Progress"
+  "so-close": "So close!",
+  "wow-almost-there": "Wow, almost there!",
+  "awesome": "Awesome!",
+  "woohoo": "Woohoo!",
+  "you-can-do-it": "You can do it!",
+  "amazing-work": "Amazing work!",
+  "nice-job": "Nice job!",
+  "keep-it-up": "Keep it up!",
+  "now-were-rolling": "Now we're rolling!",
+  "thanks-for-helping": "Thanks for helping!",
+  "making-great-progress": "Making great progress!",
+  "great-job": "Great job!",
+  "first": "first",
+  "next": "next",
+  "just": "Just",
+  "badge-type-missions": "missions",
+  "badge-type-distance": "distance",
+  "badge-type-labels": "labels",
+  "badge-type-validations": "validations",
+  "badge-missions-singular": "mission",
+  "badge-missions-plural": "missions",
+  "badge-distance-singular": "km",
+  "badge-distance-plural": "km",
+  "badge-labels-singular": "label",
+  "badge-labels-plural": "labels",
+  "badge-validations-singular": "validation",
+  "badge-validations-plural": "validations",
+  "more-unit-until-achievement": "<strong>{{n}} more $t({{unit}})</strong> until your $t({{firstOrNext}}) achievement.",
```

Comment:
Here's the sentence where I combined your different fragments of a sentence together. You can reference other translations from within a translation using `$t()` and you can get the value of a parameter using `{{}}`. So we can even pass in a translation as a parameter, and then stick that into the current translation using `$t({{}})`.